src/CommandingOfficers/Modifiers/IndirectRangeBoostModifier.java

```diff
@@ -1,43 +1,63 @@
 package CommandingOfficers.Modifiers;
 
+import java.util.ArrayList;
 import java.util.Map;
 
 import CommandingOfficers.Commander;
 import Units.UnitModel;
 import Units.Weapons.WeaponModel;
 
-public class IndirectRangeBoostModifier extends UnitRemodelModifier implements COModifier
+public class IndirectRangeBoostModifier implements COModifier
 {
+  ArrayList<WeaponModel> modelsToModify;
   private int boost;
 
   public IndirectRangeBoostModifier(int boost)
   {
-    super();
     this.boost = boost;
+    modelsToModify = new ArrayList<WeaponModel>();
   }
 
-  /** Add a new unit transform assignment. */
-  public Map<UnitModel, UnitModel> init(Commander commander)
+  /** Collects all the ranged weapon models */
+  public void init(Commander commander)
```

Comment:
I'm curious why this is a separate function instead of doing it in the constructor?

Reply:
Because historical reasons.
Probably makes sense to be rid of it, yes.